src/main/kotlin/JacocoToCoberturaPlugin.kt

```diff
@@ -65,6 +69,7 @@ class JacocoToCoberturaPlugin : Plugin<Project> {
                     kotlinSources + javaSources + customSources,
                     jacocoData
             )
+            println("Found ${kotlinSources.size} Kotlin Android Sources.")
```

Comment:
I forgot I left this debug log in, but it might be useful info. Maybe it should run through some verbose logging level, this is my first time working on a gradle plugin so I'm not really sure the right way of doing that.

Reply:
If you don't mind I included your changes in 1.1.1 PR. 
Can you confirm it works as expected as I don't have an Android project with jacoco report for testing